seed/seed.json

```diff
@@ -216,14 +268,14 @@
             "filter": {
                 "min_version": "91.1.26.37",
                 "channel": ["NIGHTLY"],
-                "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
+                "platform": ["ANDROID"]
```

Comment:
Show custom ad notifications to 60% of users on Android for Nightly build channels